docs/services/slam/cloudslam/_index.md

```diff
@@ -0,0 +1,422 @@
+
+Running `cloudslam` incurs cost for Data Management, Cloud Data Upload, and Cloud Data Egress. Currently, you incur no cost for compute.
+See Viam's [Pricing](https://www.viam.com/product/pricing) for more information.
+
+{{% /alert %}}
+
+### supported algorithms
+
+Currently cloudslam only supports the [cartographer module](../cartographer/) as a SLAM algorithm.
+
+## the SLAM Library page
+
+On the robots page in the [Viam app](https://app.viam.com/robots), change the tab from **Machines** to **SLAM Library**. From here, you can find:
+
+1.  A list of all maps generated in that location. The list shows the name and version of the map, which machine was used for mapping, and when the map was created. You can also view previous versions of a map, if a map has been updated multiple times.
+    ![offline mapping available maps](/services/slam/offline-mapping-available-maps.png)
+
+2.  You can create or update a map using a previously collected dataset by clicking the **Make new map** on the top right and specify a map name or click **Update map** next to an existing map. See [using previously captured data](#using-previously-captured-data-offline-mode) for more information on how to do this!
+
+3.  A table showing active and failed cloudslam sessions. The table highlights the name of the map, which robot is currently mapping, and whether the map is in progress or has failed. The table can also be used to stop active mapping sessions.
+    ![offline mapping maps computing table](/services/slam/offline-mapping-maps-computing-table.png)
+
+4.  You can view maps in more detail in a dynamic pointcloud viewer by selecting the `View Map` button on one
+    ![slam library view map](/services/slam/slam-library-view-map.png)
+
+5.  You can delete maps by clicking on the trash can icon in the upper right-hand corner of a map's card.
+
+## Mapping with a live machine (Online Mode)
+
+You can configure the [cloudslam-wrapper](https://github.com/viam-modules/cloudslam-wrapper) module and data capture on your machine to run SLAM in the cloud.
+
+### Requirements
+
+To use cloudslam on a live machine, the following requirements must be met:
+
+1.  A cloudslam supported algorithm must be configured on the machine. Currently this is only the [cartographer module](../cartographer/). Please configure a supported algorithm on the machine before continuing.
+
+2.  A location owner API Key or higher. See [Add an API key](/cloud/rbac/#api-keys) to learn how to create a key!
+
+### Configuration
+
+1. Enable data capture and configure your `cloudslam-wrapper` SLAM service:
+
+{{< alert title="Tip - Managing Data Capture" color="tip" >}}
+Note that [Data Capture](/services/data/capture/) continuously monitors and captures your machine’s sensor data while the machine is running. To avoid incurring charges while not in use, [turn off data capture for your sensors](/services/data/capture/) once you have finished your SLAM session.
```

Comment:
```suggestion
Note that when the [data management service](/services/data/) is enabled, it continuously monitors and syncs your machine’s sensor data while the machine is running.
To avoid incurring charges while not in use, [turn off data capture for your sensors](/services/data/capture/) once you have finished your SLAM session.
```